services/harbor/1.16.0/defaults/cm.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: harbor-1.16.0-d2iq-defaults
+  namespace: ${releaseNamespace}
+data:
+  values.yaml: |
+    ---
+    core:
+      replicas: 3
+    portal:
+      replicas: 3
+    # registry:
+    #   replicas: 3
+
+    existingSecretAdminPassword: harbor-admin-password
+
+    expose:
+      tls:
+        enabled: false
+      type: ingress
+      ingress:
+        hosts:
+          core: ""
+        className: kommander-traefik
+        annotations:
+          traefik.ingress.kubernetes.io/router.entrypoints: registry
+          traefik.ingress.kubernetes.io/router.tls: "true"
+
+    redis:
+      type: internal
+
+    database:
+      type: internal
+      internal:
+        priorityClassName: system-cluster-critical
```

Comment:
Do we need to set this? If so, what about the other components?

Reply:
I've changed this to set priority class name for `core`, `registry`, `portal` and `jobservice` (`high`).